### PR TITLE
Code 140

### PR DESCRIPTION
**Pull request by vikas199**
## Description
No Story. Small Refactor before the sprint ends

## Tests

- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by vikas199** [comment]
jenkins retest this please

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js [hunk 1]
```diff
@@ -99,6 +98,54 @@ describe('AssessmentContainerInner />', () => {
     })
   })
 
+  describe('isSubmit warning shown', () => {
+    it('isSubmitWarning is false it does not renders the ConfidentialityWarning component', () => {
+      const wrapper = shallow(<AssessmentContainerInner {...props} />)
+      wrapper.instance().setState({
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: I think `wrapper.setState()` is the same as `wrapper.instance().setState()`

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js [hunk 2]
```diff
@@ -99,6 +98,54 @@ describe('AssessmentContainerInner />', () => {
...
+      expect(WarningShow.find('ConfidentialityWarning').length).toBe(0)
+    })
+
+    it('isSubmitWarning is true it renders the ConfidentialityWarning component', () => {
+      const wrapper = shallow(<AssessmentContainerInner {...props} />)
+      wrapper.instance().setState({
+        isSubmitWarningShown: true,
+      })
+      const WarningShow = wrapper.find('WarningShow').dive()
+      expect(WarningShow.find('ConfidentialityWarning').length).toBe(1)
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: `.exists() => true/false` is sometimes more clear than checking that the length is 0 or 1.

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js [hunk 3]
```diff
@@ -99,6 +98,54 @@ describe('AssessmentContainerInner />', () => {
...
+    it('sets isSubmitWarningShown to false when handleSubmitWarning is called', () => {
+      const wrapper = shallow(<AssessmentContainerInner {...props} />)
+      wrapper.instance().setState({
+        isSubmitWarningShown: true,
+      })
+      wrapper
+        .find('WarningShow')
+        .props()
+        .handleSubmitWarning(false)
+      wrapper.instance().handleSubmitWarning(false)
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: We should not need to call this here. Calling `handleSubmitWarning` on the child should suffice.

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js [hunk 4]
```diff
@@ -99,6 +98,54 @@ describe('AssessmentContainerInner />', () => {
...
+    it('sets isSubmitWarningShown to true when handleSubmitWarning is called', () => {
+      const wrapper = shallow(<AssessmentContainerInner {...props} />)
+      wrapper.instance().setState({
+        isSubmitWarningShown: false,
+      })
+      wrapper
+        .find('WarningShow')
+        .props()
+        .handleSubmitWarning(true)
+      wrapper.instance().handleSubmitWarning(true)
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: Same here. Why are we calling it twice? We should only call it on the child.

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js (hunk 1 above)
Comment (vikas199) [reply app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: yes it is same i have console logged and checked the state

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js (hunk 2 above)
Comment (vikas199) [reply app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: ok I will make a change

File: app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js (hunk 4 above)
Comment (vikas199) [reply app/javascript/Application/components/Assessment/AssessmentContainerInner.test.js]: This one is to check whether the state is changed or not when handlesubmitwaring is invoked

**Pull request merged**